README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
```

Comment:
Using env variables and then simply `os.Getenv(...)` could even be nicer. It doesn't require putting another file in the filesystem, finding it, etc. Your run script can simply set an env variable, no matter where the contents of that come from.